chain-signatures/node/src/protocol/posit.rs

```diff
@@ -45,7 +45,7 @@ impl PositAction {
 pub enum PositInternalAction<S> {
     StartProtocol(Vec<Participant>, Positor<S>),
     Reply(PositAction),
-    Rejected,
+    Abort,
```

Comment:
why is this renamed in this PR? :sweat_smile:

Reply:
Because I promised to rename it in a different PR from this one: https://github.com/sig-net/mpc/pull/421